.github/workflows/ci.yml

```diff
@@ -60,6 +60,9 @@ jobs:
       
     - name: Create multimedia dir
       run: sudo mkdir -p /opt/multimedia && id
+
+    - name: Clone plan-evaluator with practitioner and practitioner-role models
+      run: cd .. && git clone https://github.com/OpenSRP/opensrp-plan-evaluator.git && cd opensrp-plan-evaluator && git checkout add-practitioner-and-role-models && mvn clean install -Dmaven.test.skip=true -Dmaven.javadoc.skip=true && cd ../opensrp-server-core/
```

Comment:
Why do we need this, these should be resolved by maven as they would normally be resolved at runtime

Reply:
I needed that to check the coverage and other build issues using CI. It's removed now